pkg/services/ngalert/notifier/file_store.go

```diff
@@ -29,7 +29,7 @@ func NewFileStore(orgID int64, store kvstore.KVStore, workingDirPath string) *Fi
 		workingDirPath: workingDirPath,
 		orgID:          orgID,
 		kv:             kvstore.WithNamespace(store, orgID, KVNamespace),
-		logger:         log.New("filestore", "org", orgID),
+		logger:         log.New("ngalert.notifier.file_store", orgID),
```

Comment:
let's rename it to `ngalert.notifier.alertmanager.file_store` because it is part of alertmanager

Reply:
Agreed